deploy/operator.yaml

```diff
@@ -15,7 +15,7 @@ spec:
       serviceAccountName: percona-xtradb-cluster-operator
```

Comment:
please regenerate bundle.yaml also

Reply:
@cap1984 Could you please share the steps to regenerate bundle.yaml? Thanks!